tests/Pest.php

```diff
@@ -2,9 +2,16 @@
 
 use Stancl\Tenancy\Tests\TestCase;
 
-uses(TestCase::class)->in(__DIR__);
+uses(TestCase::class)->in(...filesAndFoldersExcluding(['WithoutTenancy']));
```

Comment:
Can't we just hardcode the directories with normal tests instead of running code to scan them?

Reply:
Is it okay to hardcode folders and files (~20+)?

Reply:
Ah I thought that they're all in some folder already. You can stick with the code you have here then, and add a `// todo` to move all tests to a separate folder.